javascript/organizing_your_javascript_code/webpack.md

```diff
@@ -42,6 +42,21 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 // ...
 ```
 
+<div class="lesson-note lesson-note--tip" markdown="1">
+
+#### A note if your fonts are not loading
+
+  If you have any issues loading fonts with webpack, the issue is not with webpack, but how you declare the css rule.
+
+  Check out the answer on this question from Stack Overflow if you get stuck: [Webpack can't load fonts (ttf)](https://stackoverflow.com/questions/49877149/webpack-cant-load-fonts-ttf/68838464#68838464)
+</div>
+ <h4>A note if your fonts are not loading</h4>
```

Comment:
Not this needs moved up to [line 48](https://github.com/TheOdinProject/curriculum/pull/27300/files#diff-f8f6ec3b252f4b44fc3e5183409e9ea864e6763de0146c8e99d486105b66362dR48)

Reply:
Sorry @LexiphanicStroon my bad, this needs actually removed too 

```suggestion
```

We already have a h4 in markdown in the right place above